sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -470,12 +470,12 @@ def mask(self, cond, **kwargs):
   def dtype(self):
     return self._expr.proxy().dtype
 
+  isin = frame_base._elementwise_method('isin')
+
   @property
   def ndim(self):
     return self._expr.proxy().ndim
 
-  dtypes = dtype
-
```

Comment:
Sorry I don't have context here, why remove the dtypes?

Reply:
Yeah thats a good question, this is misleading from this diff alone. This is removing `DeferredDataFrameOrSeries.dtypes`, because Series isn't supposed to have a `dtypes` accessor (only `dtype`), and we already have an explicit one in `DeferredDataFrame`:
https://github.com/apache/beam/blob/a8cd05932bed9b2480316fb8518409636cb2733b/sdks/python/apache_beam/dataframe/frames.py#L1130-L1132

Reply:
Cool, ty